src/Modules/StatCan.OrchardCore.Hackathon/MigrationExtensions.cs

```diff
@@ -19,6 +19,12 @@ public static ContentPartDefinitionBuilder WithTeamField(this ContentPartDefinit
                     .WithDisplayName("Team")
                     .WithPosition(position)
                     .WithSettings(new ContentPickerFieldSettings() { DisplayedContentTypes = new string[] { "Team" } })
+                    
+                )
+                    .WithField("Attendance", field => field
+                    .OfType("BooleanField")
+                    .WithDisplayName("Attendance")
+                    .WithPosition("1")
```

Comment:
this part needs to be moved to the hackerPart. `WithTeamField` is used also by the Score ContentType which would add a attendance flag to each score contentItem. In this case you can use `.WithBooleanField()`